### DISCUSSION TITLE
跳过“是否删除无用文件”

### DISCUSSION THREAD

**Discussion by 123777tw** [lifegpc/bili#81] [Q&A] [answered]
问一下有什么参数或者方法可以跳过这个询问，达到无人值守的效果？

**Comment by lifegpc** [comment, marked as answer]
跑设置(`sersettings.exe`/`setsettings.py`)或者`-h`查看命令行参数（设置能做到的命令行参数都有

**Reply by 123777tw** [reply]
感谢。此外问一下“继续下载功能”是什么意思？我在改变设置后没有发现什么变化。

**Reply by 123777tw** [reply]
此外给个建议，仅得到未知参数以后提示使用-h查看帮助。
或者兼容一下-help -?之类的╮(￣▽￣)╭

**Reply by lifegpc** [reply]
-?没记错的话已经兼容
-help在GNU标准的命令行模式下会解析成-h -e -l -p

**Reply by lifegpc** [reply]
> 感谢。此外问一下“继续下载”功能是什么意思？我在改变设置后没有发现什么变化。

就是上次没下载完就继续下载。

**Reply by 123777tw** [reply]
> > 感谢。此外问一下“继续下载”功能是什么意思？我在改变设置后没有发现什么变化。
> 
> 就是上次没下载完就继续下载。

您是指断点续传？看起来好像确实是没有效果。可能每次我这边没有下载完都是出了错误导致的。

**Reply by 123777tw** [reply]
> -?没记错的话已经兼容
> -help在GNU标准的命令行模式下会解析成-h -e -l -p

感谢。之前是我的版本问题（1.3.8）
